wandb/sdk/artifacts/artifact_saver.py

```diff
@@ -105,6 +107,7 @@ def _save_internal(
         ttl_duration_seconds: Optional[int] = None,
         description: Optional[str] = None,
         aliases: Optional[Sequence[str]] = None,
+        tags: Optional[Sequence[str]] = None,
```

Comment:
Ah -- this is a general Python guideline (though I forget where it's suggested) for type hinting functions:
- type hints on argument types are as generic as possible
- type hints on return types are as specific as possible
... though admittedly this is followed in some modules more consistently than others.  It's followed here, at least 

context: this is so eg static type checkers can recognize that it's acceptable to pass a `tuple` here if needed (instead of strictly only `lists`) and the method should still work as expected